avatars/processors/inter_record_range_difference.py

```diff
@@ -185,13 +225,19 @@ def postprocess(self, source: pd.DataFrame, dest: pd.DataFrame) -> pd.DataFrame:
                 "Expected no missing values for `sort_by_variable` in source, got column with nulls instead"
             )
 
+        if self.keep_record_order and len(set(source.index) ^ set(dest.index)) > 0:
```

Comment:
```suggestion
        if self.keep_record_order and len(set(source.index).symmetric_difference(dest.index)) > 0:
```

NIT:
I think you can use `symmetric_difference` instead of the XOR operator (`^`), it makes it easier to search for for those that don't know what the XOR operator does.

https://stackoverflow.com/a/22736711